debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/strategy/AbstractConnectionConfiguration.java

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.mysql.strategy;
+
+import static io.debezium.config.CommonConnectorConfig.DATABASE_CONFIG_PREFIX;
+import static io.debezium.config.CommonConnectorConfig.DRIVER_CONFIG_PREFIX;
+
+import java.time.Duration;
+
+import io.debezium.config.CommonConnectorConfig;
+import io.debezium.config.Configuration;
+import io.debezium.config.Field;
+import io.debezium.connector.mysql.MySqlConnectorConfig;
+import io.debezium.jdbc.JdbcConfiguration;
+import io.debezium.jdbc.JdbcConnection;
+import io.debezium.util.Strings;
+
+/**
+ * @author Chris Cranford
+ */
+public abstract class AbstractConnectionConfiguration implements ConnectionConfiguration {
+
+    public static final String URL_PATTERN = "${protocol}://${hostname}:${port}/?useInformationSchema=true&nullCatalogMeansCurrent=false&useUnicode=true&characterEncoding=UTF-8&characterSetResults=UTF-8&zeroDateTimeBehavior=CONVERT_TO_NULL&connectTimeout=${connectTimeout}";
```

Comment:
Are really all the the connection parameters relevant for both MySQL and MariaDB? If not then we should IMHO spearate connections strings at this stage too.

Reply:
Likely not, I'll leave this as a todo to clean-up.